### PR TITLE
feat: clear browser on simulation restart

### PR DESCRIPTION
**Pull request by alexanderczigler**
The code is still very messy but we have introduced funcitonality to clear the browser's states when the simulator backend restarts.

The backend also listens to a reset event so frontend can build an interface where a human can force a restart.

### CODE REVIEW

File: packages/simulator/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "dev": "nodemon web/index.js",
     "test": "jest __tests__",
     "watch": "jest --watch __tests__",
-    "start": "node web/index.js"
+    "start": "nodemon web/index.js"
```
Comment (izabellalarsson) [inline packages/simulator/package.json:10]: 🙏

**Review by izabellalarsson — APPROVED** [review]

**Comment by izabellalarsson** [comment]
Maybe add a placeholder button in the UI aswell to reset from there? Or we can do that later on

**Review by mikaelaalu — APPROVED** [review]
lgtm!

**Comment by alexanderczigler** [comment]
@izabellalarsson @mikaelaalu I think it is a good idea to add the button but I would love to have this merged now and that we add the button later, as things are a bit messy right now in the code and in my head 😅 Is that fine with you?

**Comment by mikaelaalu** [comment]
merge it!

**Pull request merged**